### DISCUSSION TITLE
Someone please disable this

### DISCUSSION THREAD

**Discussion by developomp** [anyproto/contributors#46] [Localization]
![image](https://github.com/anyproto/contributors/assets/40858122/77382cdb-a53d-4f8a-8b33-60ab33460da7)

**Comment by fuksman** [comment]
@developomp, wow! I will try to find out if there is a setting for this in Crowdin. It seems that the long link made Crowdin think that the message is not written in English.

**Reply by developomp** [reply]
Actually, because I was translating Korean, it was complaining about not having enough Korean words. I bypassed the filter by doing this lmao:
![image](https://github.com/anyproto/contributors/assets/40858122/47772982-1ba4-4c41-98a5-a4212ad9d106)

**Reply by fuksman** [reply]
Interesting. Sadly, I can't find any settings related to this. 😞

**Reply by developomp** [reply]
It's best to keep this discussion open then?
